Dockerfile

```diff
@@ -35,6 +35,16 @@ FROM openjdk:8u312-slim-buster as openjdk
 FROM gcr.io/observiq-container-images/stanza-base:v1.2.2
 WORKDIR /
 
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --no-create-home \
+    --uid 10005 \
+    otel
+
+RUN mkdir /etc/otel && chown otel:otel /etc/otel
```

Comment:
Just curious if there's a reason we use `/etc/otel` over `/opt/observiq-otel-collector`? I don't mind just wondering for consistency across containerized and non container installs.

Reply:
Contrib uses /etc/otel, which might make it easier to transition to oiq. https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/otelcontribcol/Dockerfile

Reply:
Good enough reason for me.